### PR TITLE
Fix art to be ASCII

### PR DESCRIPTION
**Pull request by strager**
The GitHub description for this project is "it's just an ascii penis". However, the README.md file contains non-ASCII Unicode code points, such as ░ (U+2591 LIGHT SHADE) and ─ (U+2500 BOX DRAWINGS LIGHT HORIZONTAL).

Redraw the artwork to be ASCII.

Note to reviewers: changing the description on GitHub would work too, though I don't have a good suggestion for what the wording should be.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,3 @@
...
-░░░░░░░░░█▒░░░░▄▀───────
-▒▒▒░░░░▄▀▒░░░░▄▀────────
-▓▓▓▓▒░█▒░░░░░█▄─────────
-█████▀▒░░░░░█░▀▄────────
-█████▒▒░░░▒█░░░▀▄───────
-███▓▓▒▒▒▀▀▀█▄░░░░█──────
-▓██▓▒▒▒▒▒▒▒▒▒█░░░░█─────
-▓▓█▓▒▒▒▒▒▒▓▒▒█░░░░░█────
-░▒▒▀▀▄▄▄▄█▄▄▀░░░░░░░█───
+8===D
```
Comment (strager) [inline README.md:2]: Should I add a `~` to look like semen? The original seems to have semen (or urine). 🤷‍♀️

**Comment by strager** [comment]
Here's another option: https://www.jwz.org/blog/2013/11/drowned-in-a-sea-of-ascii-penises/

But that artwork is also lacking semen.

**Comment by genitals** [comment]
@strager  Thank you for pointing out this regrettable oversight. 

After some internal discussion, the team has decided to not merge your changes. Your updates do more closely align with the original intent of this project, however the proposed changes lack dynamism that the founders agree is essential to the long-term success of the initiative.

I've replaced the word "ascii" with "unicode" in the the repository description. We hope this is an acceptable solution.

**Comment by strager** [comment]
Sounds good to me! 👍

**Pull request closed without merging**